app/app.js

```diff
@@ -0,0 +1,21 @@
+const express = require('express');
+const app = express();
+const {
+    getTopics,
+} = require('./controllers/topics')
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics)
+
+
+app.use((err, request, response, next) => {
+
+    if(err.status && err.msg){
```

Comment:
This too is unneeded code as there are no errors from the client side to deal with (I think this is part of your 404 for an empty database test right?)

Reply:
yes it is for the 404 test